src/data/contributors.ts

```diff
@@ -83,6 +83,12 @@ const contributor = [
     instagramUsername: "dzaksssssss",
     linkedinUsername: "Dzakwan Rachman"
   },
+  {
```

Comment:
The changes here are breaking changes! Please fix it first and ensure things don't break others' code. Thank you.